akka-projection-dynamodb/src/main/resources/reference.conf

```diff
@@ -22,6 +22,12 @@ akka.projection.dynamodb {
 
     # Trying to batch insert offsets in batches of this size.
     offset-batch-size = 20
+
+	# Number of slices to read offsets simultaneously for.  The underlying Dynamo
+	# client must be able to handle (`http.max-concurrency` plus `http.max-pending-connection-acquires`)
+	# at least this number of concurrent requests.  Defaults to 1024 (all slices simultaneously),
+	# but may be reduced.
+	offset-slice-read-parallelism = 1024
```

Comment:
I agree, let's have less as default. Since it's asyncUnordered, 32 should be enough. I assume such default would work well with the defaults of the client config?

Reply:
1024 would work with the defaults (which are 50 connections and 10k pending which would likely allow for 1024 offset queries)... so the more likely scenario to need this involves tuning the client to a small connection pool and/or pending queue.

1024 matches the earlier default "all-at-once" behavior, but this is new enough that maybe the defaults can be expected to change from version to version?

Reply:
yes, I see no problem to change this behavior in next patch

Reply:
Further consideration (from spelling things out more in the comment)... the adverse impact of being smaller (e.g. 64 vs. 1024 means going from ~10ms querying for offsets to ~200ms) is far less than the impact of a projection being restarted with backoff (default is 3 seconds) due to being unable to query.